lib/rubocop/cop/layout/class_structure.rb

```diff
@@ -132,7 +132,6 @@ module Layout
       #     end
       #   end
       #
-      # @see https://rubystyle.guide#consistent-classes
```

Comment:
[memo] It was the same as the output in the `Reference`, so it has been removed here.